app/build.gradle.kts

```diff
@@ -52,4 +57,11 @@ dependencies {
     implementation(libs.kotlinx.coroutines.android)
     implementation(libs.mmkv.android)
     implementation(libs.kotlinx.serialization)
+
+    implementation("androidx.viewpager2:viewpager2:1.1.0")
+    implementation("com.tbuonomo:dotsindicator:5.0")
+
```

Comment:
add dependency in lib.toml file to centralize and use as eg implementation(libs.mmkv.android)